ast2vec/repo2/base.py

```diff
@@ -2,18 +2,19 @@
 import logging
 import multiprocessing
 import os
-from queue import Queue
 import re
 import shutil
 import subprocess
 import tempfile
 import threading
 from collections import namedtuple
+from queue import Queue
```

Comment:
Ok, there is nothing about the order of `import ...` and `from ... import ...` in PEP8.
My style feature. I change it.